### DISCUSSION TITLE
Outline not working

### DISCUSSION THREAD

**Discussion by mikeyd2017** [tailwindlabs/tailwindcss#9901] [Help]
Why does outline not work? https://tailwindcss.com/docs/outline-style#setting-the-outline-style

Outline styles are not working properly and and they're not being recognized. 

Unknown utility "outline"ts(-2020)
Unknown theme value "outline[]"ts(-2022)

**Comment by wongjn** [comment]
Could you give some more details about how it is not working? A [Tailwind Play](https://play.tailwindcss.com/) or some reproduction code would be helpful!
